docs/_docs/persistence/snapshot.adoc

```diff
@@ -0,0 +1,181 @@
+procedure. Doing this the cluster will eventually get a point in time when all previously started transactions
+finished on primary and backups, and new ones are hold until a new snapshot operation will be started.
+
+The local system files consistency (e.g. cache group partition files, binary metadata files, configuration files) achieved
+by copying them to the destination snapshot directory with tracking all concurrent ongoing changes. Tracking concurrent
+changes during copying of cache group partition files will require additional space in Ignite system directory.
+
+*Snapshot Structure.*
+
+The created snapshot has the same
+link:https://cwiki.apache.org/confluence/display/IGNITE/Ignite+Persistent+Store+-+under+the+hood#IgnitePersistentStoreunderthehood-FoldersStructure[Directory Structure]
+as the Ignite native persistence does with keeping nodes `consistentId` in snapshot directory. The `wal` directory will
+be excluded from snapshot since recovery procedures are not necessary for cache group data files.
+
+.Example of snapshot directory structure
+[source,shell]
+----
+work
+└── snapshots
+    └── backup23012020
+        ├── binary_meta
+        │   ├── snapshot_IgniteClusterSnapshotSelfTest0
+        │   ├── snapshot_IgniteClusterSnapshotSelfTest1
+        │   └── snapshot_IgniteClusterSnapshotSelfTest2
+        ├── db
+        │   ├── snapshot_IgniteClusterSnapshotSelfTest0
+        │   │   └── cache-txCache
+        │   │       ├── cache_data.dat
+        │   │       ├── part-3.bin
+        │   │       ├── part-4.bin
+        │   │       └── part-6.bin
+        │   ├── snapshot_IgniteClusterSnapshotSelfTest1
+        │   │   └── cache-txCache
+        │   │       ├── cache_data.dat
+        │   │       ├── part-1.bin
+        │   │       ├── part-5.bin
+        │   │       └── part-7.bin
+        │   └── snapshot_IgniteClusterSnapshotSelfTest2
+        │       └── cache-txCache
+        │           ├── cache_data.dat
+        │           ├── part-0.bin
+        │           └── part-2.bin
+        └── marshaller
+----
+
+*Requirements.*
+
+You need an extra space on disk for the configured Ignite native persistence directory up to the current size this directory.
+This space will be used for storing intermediate snapshot results and will be freed when the snapshot operation ends.
+
+*Limitations.*
+
+The snapshot procedure has some limitations that you should be aware of before using it at your production environment:
+
+* taking a snapshot of particular cache or cache group not supported;
+* in-memory caches will not be included into snapshot;
+* encrypted caches not supported and will be ignored;
+* only single snapshot operation can be started at once;
+* snapshot procedure will be interrupted if any baseline node leave the cluster;
+* snapshot recovery at the same baseline topology with the same consistent node id;
+
+== Configuration
+
+You may configure the destination snapshot directory via `IgniteConfiguration`.
```

Comment:
Fixed like this one 
{code}
The destination snapshot directory can be configured via `IgniteConfiguration`.
{code}
since only destination directory can be configured through configuration.